packages/embark-ens/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@babel/runtime-corejs2": "7.3.1",
-    "async": "^2.6.1",
+    "async": "2.6.1",
```

Comment:
Could be `2.6.x` too but that's fine

Reply:
At present we pretty consistently use exact versions for any dependencies other than our own monorepo packages (`embark-*`), and also the same version for a given package across members of the monorepo.